model/post.go

```diff
@@ -170,7 +171,7 @@ func (o *Post) IsValid() *AppError {
 
 	if !(o.Type == POST_DEFAULT || o.Type == POST_JOIN_LEAVE || o.Type == POST_ADD_REMOVE ||
```

Comment:
This conditional statement is not easy to read. Would it be possible to change, somewhat like below?
```
	switch o.Type {
	case
		POST_DEFAULT,
		POST_JOIN_LEAVE,
		POST_ADD_REMOVE,
		POST_JOIN_CHANNEL,
		POST_LEAVE_CHANNEL,
		POST_LEAVE_TEAM,
		POST_REMOVE_FROM_CHANNEL,
		POST_ADD_TO_CHANNEL,
		POST_ADD_TO_TEAM,
		POST_SLACK_ATTACHMENT,
		POST_HEADER_CHANGE,
		POST_PURPOSE_CHANGE,
		POST_DISPLAYNAME_CHANGE,
		POST_CHANNEL_DELETED:
	default:
		if !strings.HasPrefix(o.Type, POST_CUSTOM_TYPE_PREFIX) {
			return NewAppError("Post.IsValid", "model.post.is_valid.type.app_error", nil, "id="+o.Type, http.StatusBadRequest)
		}
	}
```

Reply:
Hi @saturninoabril , gonna change it.